### ISSUE TITLE
Questions about second stage

### ISSUE THREAD

**Issue by gaojunbin** [tianweiy/CenterPoint#346]
Hi tianwei,

I am confusing for the sencond stage in roi (https://github.com/tianweiy/CenterPoint/blob/8fcdc944bbb455bd25943b331aaf961aa0ab32cd/det3d/models/roi_heads/roi_head_template.py#L175). I found you predict offset by fc layer and directly add it to the first stage prediction result. But offset can be negative. For bbox size(w,h,l), if you add a very small negative number, the final refine bbox size will also be negative, is this reasonable. I found in the first stage, the size is sent to a torch.exp function to make it greater than 0. 

Looking forward to your reply, thanks!

**Comment by tianweiy** [comment]
yeah, I see the problem. here we follow previous works and just predict the absolute offset. It seems to work generally well in practice as the offset is learned to be a near zero value.
